### DISCUSSION TITLE
Add google tag manager

### DISCUSSION THREAD

**Discussion by lowbits** [Teranode/nuxt-module-alternatives#69] [Ideas]
Google Tag Manager is probably a highly used module from nuxt2, I think the development for nuxt3 will start not soon.
Maybe we can add an alternative here, does it makes sense?
Should I start with it?

**Comment by Denoder** [comment]
ill take a look at gtm

**Comment by Denoder** [comment]
wouldn't vue-gtm suffice as a possible replacements simply requiring you to create a plugin yourself?
https://github.com/gtm-support/vue-gtm

**Reply by lowbits** [reply]
Yes, I will try to go with this approach, but it's not working on SSR.
Maybe currently I don't need SSR.
Thank you for checking 🤙
